File: maple/maple-type-interface/src/main/java/com/maple/service/UserJobService.java
package com.maple.service;

import com.maple.pojo.Userjob;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface UserJobService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Userjob> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Userjob userjob);
	
	
	/**
	 * 修改
	 */
	public void update(Userjob userjob);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Userjob findOne(int id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(int[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Userjob userjob, int pageNum, int pageSize,String name);

	public PageResult findUserPage(Userjob userjob, int page, int rows,String name);

    void updateUserJobStatus(int jobid, String userid, String createuserid);
}

File: maple/maple-user-web/src/main/webapp/js/service/userjobService.js
//服务层
app.service('userjobService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../userjob/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../userjob/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../userjob/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../userjob/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../userjob/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../userjob/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../userjob/search.do?page='+page+"&rows="+rows, searchEntity);
	}
    //搜索
    this.searchUser=function(page,rows,searchEntity){
        return $http.post('../userjob/searchUser.do?page='+page+"&rows="+rows,searchEntity);
    }
    this.resumemodel=function(userid){
        return $http.get('../user/findResumeModel.do?userid='+userid);
    }
    this.findOneJob=function(id){
        return $http.get('../job/findOne.do?id='+id);
    }
    this.updateUserJobStatus=function (jobid,userid) {
		return $http.get('../userjob/updateUserJobStatus.do?jobid='+jobid+"&userid="+userid);
    }
});
